feat-content.html

```diff
@@ -10,26 +10,25 @@ <h4 class="feat-content-title">Bioinformatics Community Conference 2020</h4>
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="https://www.youtube.com/watch?v=1JWOVGzzgMc&feature=youtu.be" >
-            <h4 class="feat-content-title"> Introduction to Dockstore Webinar</h4>
-            <p class="feat-card-description">A simple overview of Dockstore. Covers the fundamentals of finding workflows and provides examples of running analysis with sample data.
-            </p>
+           href="https://biodatacatalyst.nhlbi.nih.gov/fellows/program/">
+            <h4 class="feat-content-title">Apply Now: BioData Catalyst Fellowship</h4>
+            <p class="feat-card-description">The NHLBI BioData Catalyst Fellows Program provides early-career researchers funding and support for novel innovative data science and data-focused research problems. Due June 26, 2020.</p>
         </a>
     </div>
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="https://docs.dockstore.org/en/develop/advanced-topics/snapshot-and-doi.html" >
-            <h4 class="feat-content-title">Creating Snapshots and Requesting DOIs</h4>
-            <p class="feat-card-description">Learn how to create snapshots and Digital Object Identifiers (DOIs) for workflows using Zenodo.</p>
+           href="organizations/BroadInstitute/collections/pgs">
+            <h4 class="feat-content-title">Viral Analysis Workflows for COVID-19</h4>
+            <p class="feat-card-description">A collection of viral NGS workflows for analyzing COVID-19. Learn how to use them with your own data in linked Terra workspaces from the Broad Viral Genomics & Data Sciences Platform.</p>
         </a>
     </div>
 
     <div class="feat-content-card">
         <a class="cardClick" target="_blank" rel="noopener noreferrer"
-           href="organizations/bdcatalyst">
-            <h4 class="feat-content-title">NHLBI BioData Catalyst</h4>
-            <p class="feat-card-description">A biomedical data platform leveraging emerging opportunities in data science to open new frontiers in heart, lung, blood, and sleep (HLBS) research.</p>
+           href="https://www.youtube.com/watch?v=1JWOVGzzgMc&feature=youtu.be" >
```

Comment:
TIL `feature=youtu.be`

Reply:
Not sure what it does, never noticed it before. Works the same with and without (gah had to hear myself to check it).